multinode-demo/fullnode.sh

```diff
@@ -286,10 +351,10 @@ while true; do
 
     done
 
-    echo "############## New genesis detected, restarting fullnode ##############"
+    echo "############## New genesis detected, restarting... ##############"
```

Comment:
heh, now we can do this: `echo "############## New genesis detected, restarting $node_type  ##############"`